doc/man/dune-build.1

```diff
@@ -0,0 +1,346 @@
+
+       --diff-command=VAL (absent DUNE_DIFF_COMMAND env)
+           Shell command to use to diff files. Use - to disable printing the
+           diff.
+
+       --disable-promotion (absent DUNE_DISABLE_PROMOTION env)
+           Disable all promotion rules
+
+       --display=MODE
+           Control the display mode of Dune. See dune-config(5) for more
+           details.
+
+       --dump-memo-graph=FILE
+           Dumps the dependency graph to a file after the build is complete
+
+       --dump-memo-graph-format=FORMAT (absent=gexf)
+           File format to be used when dumping dependency graph
+
+       --dump-memo-graph-with-timing
+           With --dump-memo-graph, will re-run each cached node in the Memo
+           graph after building and include the runtime in the output. Since
+           all nodes contain a cached value, this will measure just the
+           runtime of each node
+
+       --help[=FMT] (default=auto)
+           Show this help in format FMT. The value FMT must be one of auto,
+           pager, groff or plain. With auto, the format is pager or plain
+           whenever the TERM env var is dumb or undefined.
+
+       --ignore-promoted-rules
+           Ignore rules with (mode promote), except ones with (only ...). The
+           variable %{ignoring_promoted_rules} in dune files reflects whether
+           this option was passed or not.
+
+       --instrument-with=BACKENDS (absent DUNE_INSTRUMENT_WITH env)
+           "Enable instrumentation by BACKENDS. BACKENDS is a comma-separated
+           list of library names, each one of which must declare an
+           instrumentation backend.
+
+       -j JOBS
+           Run no more than JOBS commands simultaneously.
+
+       --no-buffer
+           Do not buffer the output of commands executed by dune. By default
+           dune buffers the output of subcommands, in order to prevent
+           interleaving when multiple commands are executed in parallel.
+           However, this can be an issue when debugging long running tests.
+           With --no-buffer, commands have direct access to the terminal.
+           Note that as a result their output won't be captured in the log
+           file. You should use this option in conjunction with -j 1, to
+           avoid interleaving. Additionally you should use --verbose as well,
+           to make sure that commands are printed before they are being
+           executed.
+
+       --no-config
+           Do not load the configuration file
+
+       --no-print-directory
+           Suppress "Entering directory" messages
+
+       --only-packages=PACKAGES
+           Ignore stanzas referring to a package that is not in PACKAGES.
+           PACKAGES is a comma-separated list of package names. Note that
+           this has the same effect as deleting the relevant stanzas from
+           dune files. It is mostly meant for releases. During development,
```

Comment:
```suggestion
           `dune` files. It is mostly meant for releases. During development,
```

These backticks might not be helpful in a `.1` file, but `dune` should be in monospace when talking about `dune` files. 

Let me know how to format monospace in a `.1` file, and I’ll go back through this with my suggestions. I couldn’t find how to do it online.